===================================================================
RCS file: /home/cvs/cvsroot/abiword-plugins/wp/impexp/pdf/xp/ie_imp_PDF.cpp,v
retrieving revision 1.2
retrieving revision 1.3
diff -u -r1.2 -r1.3
--- abiword-plugins/wp/impexp/pdf/xp/ie_imp_PDF.cpp	2005/06/29 02:22:13	1.2
+++ abiword-plugins/wp/impexp/pdf/xp/ie_imp_PDF.cpp	2006/01/22 01:24:52	1.3
@@ -75,7 +75,7 @@
       UT_tmpnam(temporary_file);
 
       TextOutputDev *textOut = new TextOutputDev(temporary_file, gFalse, gFalse, gFalse);
-      doc->displayPages(textOut, 1, doc->getNumPages(), 72, 72, 0, gTrue, gFalse);
+      doc->displayPages(textOut, 1, doc->getNumPages(), 72, 72, 0, gTrue, gFalse, gFalse);
       delete textOut;
 
       IE_Imp_Text txt_imp(getDoc(), "UTF-8");      

